todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksFragment.java

```diff
@@ -403,12 +415,16 @@ public View getView(int i, View view, ViewGroup viewGroup) {
 
             // Active/completed task UI
             completeCB.setChecked(task.isCompleted());
-            if (task.isCompleted()) {
-                rowView.setBackgroundDrawable(viewGroup.getContext()
-                        .getResources().getDrawable(R.drawable.list_completed_touch_feedback));
+
+            if (selectedItem != null && selectedItem.equals(task.getId())) {
+                //noinspection deprecation
+                rowView.setBackgroundDrawable(ContextCompat.getDrawable(
+                        viewGroup.getContext(), R.drawable.selectedTask));
+
             } else {
-                rowView.setBackgroundDrawable(viewGroup.getContext()
-                        .getResources().getDrawable(R.drawable.touch_feedback));
+                //noinspection deprecation
+                rowView.setBackgroundDrawable(ContextCompat.getDrawable(
```

Comment:
it's not part of this PR but I feel like we should be doing a better job at view states than this